assets/src/stories-editor/helpers/index.js

```diff
@@ -519,15 +519,30 @@ export const renderStoryComponents = () => {
 		}
 	}
 
+	/**
+	 * Gets a component that conditionally displays, depending on whether the pages are reordering.
+	 *
+	 * @param {Function} component The component to conditionally display.
+	 * @return {Function|null} The passed omponent, or null.
+	 */
+	const getConditionalDisplay = ( component ) => {
```

Comment:
We already have a `withIsReordering` higher-order component.

I would suggest removing `getConditionalDisplay` in favor of using `withIsReordering` together with the [`ifCondition`](https://github.com/WordPress/gutenberg/blob/a6c36e53974ac8a9fdd6163e61b54064cfd8910f/packages/compose/src/if-condition/README.md) HOC offered by Gutenberg. This here is a perfect use case for that.

As for the `render()` part, I would probably put the `ifCondition` and `withIsReordering` within a `compose` in `assets/src/components/shortcuts/index.js` and `assets/src/components/inserter/index.js`

Reply:
Hi @swissspidy, thanks for pointing that out. I'll apply your suggestions, but it might be a while if that's alright 😄

Reply:
I've applied the changes now in cff2ec8f6cd57588d7edb9e86ec38579aa90d0ee